app/ldap.go

```diff
@@ -19,7 +18,8 @@ func (a *App) SyncLdap() {
 			if ldapI := a.Ldap; ldapI != nil {
 				ldapI.StartSynchronizeJob(false)
 			} else {
-				mlog.Error(fmt.Sprintf("%v", model.NewAppError("SyncLdap", "ent.ldap.disabled.app_error", nil, "", http.StatusNotImplemented).Error()))
+				err := model.NewAppError("SyncLdap", "ent.ldap.disabled.app_error", nil, "", http.StatusNotImplemented)
+				mlog.Error("Not executing ldap sync because ldap is not available", mlog.Err(err))
```

Comment:
Do we really need to craft an error for here? It doesn't contain any additional information.

Reply:
Log messages aren't translated, errors are, I was trying to keep it as less intrusive as possible

Reply:
To me this log message is written for admins, which don't need translated error messages.

Reply:
fair point, I going to remove the AppError creation there